.github/workflows/refresh-licenses.yml

```diff
@@ -0,0 +1,41 @@
+# Refreshes the licences directory and creates a PR if there are any changes
+
+name: refresh-licenses
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "30 0 * * *"
```

Comment:
Probably this should be just once a week, probably once a month

Reply:
I've set it to run once a week, every Monday At 00:30.